### PR TITLE
docs: Update concept map to include all Metrics components

### LINKED ISSUE #1347: Update concept map to include cloud functions and metrics CLI
**Description**
Update concept map to include cloud functions (green, dart) and metrics CLI as a standalone installer component.

**Acceptance Criteria**
- [ ] Concept map now includes cloud functions and metrics CLI.

**Additional context**
[Concept map](https://github.com/platform-platform/monorepo/blob/master/concept_map.puml)

### PR DESCRIPTION
**Pull request by solid-vovabeloded**
Fixes #1347

## Checklist

- [x] Documented

### CODE REVIEW

File: concept_map.puml [hunk 1]
```diff
@@ -26,22 +26,29 @@ cloud "GitHub Actions" as GA
...
 rectangle "CI integrations" as CI
+rectangle "Dart Cloud Functions" as DartCloudFunctions
 
 together {
     node Firebase {
         cloud Hosting
         database Firestore
         cloud Authentication
+        cloud "Cloud Functions" as CloudFunctions
+        cloud Analytics
```
Comment (solid-vladlaba) [inline concept_map.puml:37]: What do you think if we reduce some "clouds" from the concept map? For example, by making CIs in the above and Sentry as `node` or something else.

File: concept_map.puml [hunk 2]
```diff
@@ -26,22 +26,29 @@ cloud "GitHub Actions" as GA
...
 
 rectangle "CI integrations" as CI
+rectangle "Dart Cloud Functions" as DartCloudFunctions
 
 together {
     node Firebase {
         cloud Hosting
         database Firestore
         cloud Authentication
+        cloud "Cloud Functions" as CloudFunctions
```
Comment (solid-vladlaba) [inline concept_map.puml:36]: Generally speaking, the Cloud Functions and Dart Cloud Functions are the same entity for this diagram. What if we inline it or notice somehow that our functions are written in Dart?
```suggestion
        cloud "Dart Cloud Functions" as DartCloudFunctions
```
We can make this cloud green noticing that this is part of Metrics components.

File: concept_map.puml [hunk 3]
```diff
@@ -53,12 +60,20 @@ CI -[hidden]down-> Hosting
 Hosting -[hidden]right-> Authentication
 Authentication -[hidden]right-> Firestore
 
-Firebase -[hidden]right-> Core
+Web --up--> Firestore : reads metrics and \l stores app config
+Web --up--> Authentication : authenticates
+Web --up--> Hosting : deployed on
+Web --up--> Analytics : reports analytics data
+Web -down-> Sentry : reports about errors
```
Comment (solid-vladlaba) [inline concept_map.puml:67]:
```suggestion
Web -down-> Sentry : logging
```
Actually, Sentry may be used not for errors only so maybe we can write logging here.

File: concept_map.puml [hunk 4]
```diff
@@ -53,12 +60,20 @@ CI -[hidden]down-> Hosting
...
+
+DartCloudFunctions -up-> CloudFunctions : deployed on
+DartCloudFunctions -[hidden]-> Core : uses
+DartCloudFunctions -up-> Firestore : creates aggregation \l documents
```
Comment (solid-vladlaba) [inline concept_map.puml:71]: Consider the above suggestion with `CloudFunctions`. Also, it seems like the aggregation message reveals some unnecessary details here. So, how about that?
```suggestion
DartCloudFunctions -[hidden]-> Core : uses
```

File: concept_map.puml [hunk 5]
```diff
@@ -53,12 +60,20 @@ CI -[hidden]down-> Hosting
...
 
-Web -up-> Firestore : reads metrics and \l stores app config
-Web -up-> Authentication : authenticates
-Web -up-> Hosting : deployed on
+CLI --> Web : deploys
```
Comment (solid-vladlaba) [inline concept_map.puml:73]: Not sure the CLI deploys the Web only. Instead, what if we rename CLI into `Deploy CLI` on the diagram? Then, keep the `CLI --> Core : uses` but remove the `deploys` line:
```suggestion
```
What do you think?

**Review by solid-vladlaba — COMMENTED** [review]
Well done, @solid-vovabeloded! I've added a couple of suggestions below - please let me know what you think! The new concept map diagram looks great 👍

**Review by illia-romanenko — APPROVED** [review]
LGTM - thanks @solid-vovabeloded and @solid-vladlaba. Thanks for all of the work you put to polish it! Also, FYI created https://github.com/platform-platform/monorepo/issues/1354 to improve it even further.

**Pull request merged**